action.yml

```diff
@@ -26,6 +26,6 @@ inputs:
     required: false
 
 runs:
-  using: 'node16'
+  using: 'node20'
```

Comment:
We can't just bump to `node20`, we also need to align dev dependencies and update our toolkit.

Also I don't know why GitHub made a such drastic bump and didn't update to the latest LTS node 18 instead: https://nodejs.dev/en/about/releases/

![image](https://github.com/docker/login-action/assets/1951866/b9d8ae55-eb5c-4cb9-8ab1-c4a4af769d91)

Some users already complain: https://github.com/actions/checkout/commit/97a652b80035363df47baee5031ec8670b8878ac

Anyway I will look into it and keep you posted.

Reply:
> We can't just bump to node20, we also need to align dev dependencies and update our toolkit.

Noted, thanks. I will close my other PRs too.

> Also I don't know why GitHub made a such drastic bump and didn't update to the latest LTS node

Same here

Reply:
Fyi this is tracked here https://github.com/docker/login-action/pull/593